Readme.md

```diff
@@ -9,7 +9,9 @@
 * Update the environments needed in the [default.env](./default.env) file. After you create them just run: `source default.env`
 * Run installation with `yarn`
 
-## Running tests
+## Running Tests Locally
+Before runing tests, you can modify the file [softwareTemplates.json](softwareTemplates.json) manually to meet your RHTAP environment. For instance, if your RHTAP environment is using Github as Git repository and Jenkins as Pipeline provider, you can set them as `true`.  The both properties `pipeline.version` and `priority` are not in use, we leave them here for future use.
+
```

Comment:
Just running `yarn test` could run the test, or we need to run `node generateTemplatesConfig.js` as well before `yarn test`